include/boost/process/v2/detail/impl/exe.ipp

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2022 Klemens D. Morgenstern
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+#ifndef BOOST_PROCESS_V2_IMPL_DETAIL_EXE_IPP
+#define BOOST_PROCESS_V2_IMPL_DETAIL_EXE_IPP
+
+#include <boost/process/v2/detail/exe.hpp>
+
+BOOST_PROCESS_V2_BEGIN_NAMESPACE
+
+namespace detail
+{
+
+namespace ext
+{
+
+#if defined(_WIN32)
+
+BOOST_PROCESS_V2_DECL
+HANDLE open_process_with_debug_privilege(pid_type pid) {
+    HANDLE proc = nullptr;
+    HANDLE hToken = nullptr;
+    LUID luid;
+    TOKEN_PRIVILEGES tkp;
+    if (OpenProcessToken(GetCurrentProcess(), TOKEN_ADJUST_PRIVILEGES | TOKEN_QUERY, &hToken)) {
```

Comment:
The error gets recorded to GetLastError(). Although the website doesn't seem to say why it would ever error, so I'm not sure what the failure cases would be without debugging it hands on.

Reply:
I think i know what you want now, you want me to add an error code argument to the helper functions so i can pass the errors on to the functions which call them. makes sense now. Taking care of it.